internal/handlers/pg/msg_getparameter.go

```diff
@@ -25,25 +26,80 @@ import (
 
 // MsgGetParameter OpMsg used to get parameter.
 func (h *Handler) MsgGetParameter(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	document, err := msg.Document()
+	cmd, err := msg.Document()
 	if err != nil {
 		return nil, lazyerrors.Error(err)
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/449
+	getParameter, err := cmd.Get("getParameter")
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
 
-	common.Ignored(document, h.l, "comment")
+	// SELECT * FROM 'admin'
+	resDB := types.MustNewDocument(
+		"acceptApiVersion2", false,
+		"authSchemaVersion", int32(5),
+		"quiet", false,
+		"ok", float64(1),
+	)
 
 	var reply wire.OpMsg
+	var errNoFound error
+	resDoc := resDB
+	if getParameter != "*" {
+		resDoc, errNoFound, err = selectParam(cmd, resDB)
+		if err != nil {
+			return nil, lazyerrors.Error(err)
+		}
+	}
+
 	err = reply.SetSections(wire.OpMsgSection{
-		Documents: []*types.Document{types.MustNewDocument(
-			"quiet", false,
-			"ok", float64(1),
-		)},
+		Documents: []*types.Document{resDoc},
 	})
 	if err != nil {
 		return nil, lazyerrors.Error(err)
 	}
 
-	return &reply, nil
+	comment, err := cmd.Get("comment")
+	if err == nil {
+		common.Ignored(cmd, h.l, fmt.Sprint(comment))
```

Comment:
There is not reason to make it that complicated, with `if`, error handling, etc. The old line of code is good

Reply:
that is, we ignore the comment from command?

Reply:
We ignore comment either way, aren't we? That's what `common.Ignored` does